src/ATen/native/xpu/sycl/ActivationHardshrinkKernels.cpp

```diff
@@ -0,0 +1,36 @@
+#include <ATen/OpMathType.h>
+#include <ATen/native/TensorIterator.h>
+#include <comm/xpu_aten.h>
```

Comment:
Do we need this here, as @ZhiweiYan-96 explained, it is a WA which is not elegant, and doesn't aligned CUDA